modules/webgl/src/classes/texture.js

```diff
@@ -195,14 +195,23 @@ export default class Texture extends Resource {
         type: this.type,
         dataFormat: this.dataFormat,
         border: this.border,
-        mipmaps: false
+        mipmaps: this.mipmaps
       });
     }
     return this;
   }
 
   // Call to regenerate mipmaps after modifying texture(s)
   generateMipmap(params = {}) {
+    if (!this.mipmaps) {
```

Comment:
Nit: We should have logged NPOT warning already during initialize/resize, and not needed here. 
```
if (!this.mipmaps) {
  log.warn('generateMipmap: mipmaps is not enabled.');
  return this;
}